tools/ppa-upload.sh

```diff
@@ -0,0 +1,148 @@
+#!/bin/bash
```

Comment:
Worried this will get out of sync with the mir version... Wonder if we need a... yeah, I know... git submodule with common CI bits?

Reply:
I agree it would be better to have any potentially shared CI integration in a separate source tree.

Does that have to be integrated via a submodule though? AIUI this would require having to sync multiple projects to the latest commit.

For CI integration it would be better to always download the current version.